swiftpkg/bzlmod/swift_deps.bzl

```diff
@@ -48,6 +48,13 @@ def _declare_pkgs_from_package(module_ctx, from_package, config_pkgs):
     direct_dep_repo_names = []
     direct_dep_pkg_infos = {}
     for dep in pkg_info.dependencies:
+        # Ignore unresolved dependencies, for example for a new packgage added
+        # to the `Package.swift` which has not been resolved yet.
+        # By ignoring these for now we can allow the build to progress while expecting a resolution in the future.
+        if not dep.file_system and \
+           not dep.source_control:
+            continue
```

Comment:
Should we print something saying that we detected this scenario?

Reply:
Yeah good call, this should be a warning probably. I can get that added. 

Would we want to pull in a copy of: https://github.com/MobileNativeFoundation/rules_xcodeproj/blob/44b6f046d95b84933c1149fbf7f9d81fd4e32020/xcodeproj/internal/logging.bzl#L24 this way we can sort of match how Bazel warnings look

Reply:
I am not familiar with the module. Is it smart enough to not add the colorization if it is not being output to a TTY?

Reply:
Hmm probably not it's just using print afaik

Reply:
I don't think that we should litter CI logs with color codes, if we can avoid it. For now, let's just print something. Perhaps we can come up with a Starlark logging module that can detect when to colorize or at least be configured not to colorize. We can add it to `bazel-contrib`.

Reply:
Added a `print` warning here